checkout_session.go

```diff
@@ -0,0 +1,134 @@
+package stripe
+
+import (
+	"encoding/json"
+)
+
+// CheckoutSessionDisplayItemType is the list of allowed values for the display item type.
+type CheckoutSessionDisplayItemType string
+
+// List of values that CheckoutSessionDisplayItemType can take.
+const (
+	CheckoutSessionDisplayItemTypeCustom CheckoutSessionDisplayItemType = "custom"
+	CheckoutSessionDisplayItemTypePlan   CheckoutSessionDisplayItemType = "plan"
+	CheckoutSessionDisplayItemTypeSKU    CheckoutSessionDisplayItemType = "sku"
+)
+
+// CheckoutSessionLineItemParams is the set of parameters allowed for a line item
+// on a checkout session.
+type CheckoutSessionLineItemParams struct {
+	Amount      *int64    `form:"amount"`
+	Currency    *string   `form:"currency"`
+	Description *string   `form:"description"`
+	Images      []*string `form:"images"`
+	Name        *string   `form:"name"`
+	Quantity    *int64    `form:"quantity"`
+}
+
+// CheckoutSessionPaymentIntentDataTransferDataParams is the set of parameters allowed for the
+// transfer_data hash.
+type CheckoutSessionPaymentIntentDataTransferDataParams struct {
+	Destination *string `form:"destination"`
```

Comment:
Should this also have `Amount`?

Reply:
No `amount` is gated in theory (and there for PI because it was ungated before launch)

Reply:
Ah, got it. Okay, thanks!